docker-compose.yml

```diff
@@ -5,7 +5,7 @@ version: "3"
 services:
   master:
     container_name: "${COMPOSE_PROJECT_NAME:-citus}_master"
-    image: "citusdata/citus:10.1.2"
+    image: "citusdata/citus:10.2.1"
```

Comment:
You need to update `services.worker.image` property as well

Reply:
We do not have the images ready yet. So I do not think that this `docker-compose.yml` will work now

Reply:
Fixed that one as well